tcms/core/ajax.py

```diff
@@ -161,59 +161,17 @@ def form(request):
     return HttpResponse(html())
 
 
-def tags(request, template_name="management/get_tag.html"):
-    """Get tags for test plan or test case"""
+def tags(request):
+    """ Get tags for TestPlan, TestCase or TestRun """
 
-    class Objects(object):
-        __all__ = ['plan', 'case', 'run']
-
-        def __init__(self, request, template_name):
-            self.template_name = template_name
-            for obj_type in self.__all__:
-                if request.GET.get(obj_type):
-                    self.object = obj_type
-                    self.object_pk = request.GET.get(obj_type)
-                    break
-
-        def get(self):
-            func = getattr(self, self.object)
-            return func()
-
-        def plan(self):
-            return self.template_name, TestPlan.objects.get(
-                pk=self.object_pk)
-
-        def case(self):
-            return self.template_name, TestCase.objects.get(
-                pk=self.object_pk)
-
-        def run(self):
-            return 'run/get_tag.html', TestRun.objects.get(
-                pk=self.object_pk)
-
-    class TagActions(object):
-        __all__ = ['add', 'remove']
```

Comment:
Did you delete these by mistake or you had something in mind? Can you explain?

Reply:
They are not used anywhere, so I supposed they were leftovers from some previous implementation.

Reply:
:+1: